src/img/index.js

```diff
@@ -0,0 +1,6 @@
+export {default as g_bw} from "./g_bw.jpeg";
+export {default as j_color} from "./j_color.jpeg";
+export {default as jg_bw} from "./J&G_0147.jpeg";
+export {default as jg_rings} from "./jg_rings.JPG";
+export {default as heroImg} from "./heroImg.jpg";
```

Comment:
You should come up with some kind of consistent naming conversion for your assets here. I have no opinion on them being camel or snake cased, but you should pick one and stick to it